ios/Surveys/Sources/Domain/KotlinSwift/UseCases/GetSurveysUseCase.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  GetSurveysUseCase.swift
+//  Surveys
+//
+//  Created by Su Ho on 21/11/2022.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import Combine
+import Factory
+import KMPNativeCoroutinesCombine
```

Comment:
just concern, could we remove these imports?

Reply:
Unfortunately, we can't. 🙈
Combine: AnyPublisher
Factory: @Injected
KMP: createPublisher
Shared: Shared.GetSurveysUseCase